build.gradle

```diff
@@ -114,7 +116,9 @@ dependencies {
     implementation "com.example.universe.simulator:common:${commonVersion}"
     implementation 'org.springframework.boot:spring-boot-starter-amqp'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-graphql'
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    compileOnly "com.graphql-java:graphql-java-extended-scalars:${graphqlJavaExtendedScalarsVersion}"
```

Comment:
First of all, go to your application `Run/Debug Configurations` and remove the `Add dependencies with "provided" scope to classpath` option:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/16168235/178914837-055c37ec-e929-416e-833d-e36f910b4edb.png">

After this, if you run the app, it will fail to start because of the `java.lang.ClassNotFoundException: graphql.scalars.ExtendedScalars` error. So you need to change the scope of the `graphql-java-extended-scalars` dependency to `implementation`. `compileOnly` worked in your IDE, but would have failed in production, because the `Add dependencies with "provided" scope to classpath` option only applies in the context of your IDE. CI tests didn't fail because you have this depedency additionally declared as `testRuntimeOnly`.

Action points:
* Change the scope of the dependency to implementation
* Remove the testRuntimeOnly scope of the dependency

Reply:
Done.